libs/common/src/errors/index.ts

```diff
@@ -12,3 +12,4 @@ export * from './invalid-facility-id.error';
 export * from './deal-version.error';
 export * from './invalid-parameter-error';
 export * from './invalid-payload-error';
+export * from './cancellation-not-found.error';
```

Comment:
`deal-cancellation-not-found.error`?

Incase we add "cancellation" features to something else